### PR TITLE
Android on ubuntu takeover/engage page - due 14 August 2019

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Developing android on ubuntu takeover, engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- see that:
    - it is on the /engage page - http://0.0.0.0:8001/engage
    - the takeover looks correct and goes to the right engage page here is the [copy doc](https://docs.google.com/document/d/10w-f6FEEYGPGcrFFPiZmWioZnfC3lfvfjqx_tMsi1nE/edit) - http://0.0.0.0:8001/engage/developing-android-on-ubuntu here is the [brief](https://docs.google.com/spreadsheets/d/16tDtjCc9V29__YF5-5NTOkdX2Xp5sMXniaMRtNP-h_g/edit#gid=2113339190)
    - that the engage page copy, meta data and form are correct (*note the download isn't on the thank you form*)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1496

### takeover
image

### engage page
image

### engage overview
image

### CODE REVIEW

File: templates/engage/developing-android-on-ubuntu.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{% extends_with_args "engage/base_engage.html" with title="A guide to developing Android apps on Ubuntu" meta_image="dab160e2-anroid_on_ubuntu-social.png" meta_description="Android is the most popular mobile operating system and is continuing to grow its market share.  IDC expects that Android will have 85.5% of the market by 2022, demonstrating that app development on Android will continue to be an in-demand skill." %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1xyNrWYsYcvux_gYtoWw39XD9cP4QmiAYJnDlLuV8CvE{% endblock meta_copydoc %}
```
Comment (solazio) [inline templates/engage/developing-android-on-ubuntu.html]: this link is for a different engage page.

File: templates/engage/developing-android-on-ubuntu.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          <li class="p-list__item is-ticked">
+            How to configure and install Android Studio as a snap to get started
+          </li>
+          <li class="p-list__item is-ticked">
+            A step by step guide on creating an Android app on Ubuntu targeting an array of devices
+          </li>
+        </ul>
+      </div>
+      <div class="col-5" id="register-section">
+        {% include "engage/shared/_en_engage_form.html" with id="3340" returnURL="https://pages.ubuntu.com/Whitepaper_AndroidonUbuntu-TY.html" %}
```
Comment (solazio) [inline templates/engage/developing-android-on-ubuntu.html]: id should be 3440 according to this https://docs.google.com/spreadsheets/d/16tDtjCc9V29__YF5-5NTOkdX2Xp5sMXniaMRtNP-h_g/edit#gid=877682240

File: templates/engage/developing-android-on-ubuntu.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+      .p-takeover--android-on-ubuntu .p-takeover__title {
+        font-weight: 100;
+      }
+
+      @media (min-width: 768px) {
+        .p-takeover--android-on-ubuntu {
+          background-color: #2C001E;
+          background-image: linear-gradient(199deg, #E95420 0%, #5E2750 25%, #2C001E 50%);
+          background-image: url('{{ ASSET_SERVER_URL }}1487c2c6-suru-background.svg');
```
Comment (solazio) [inline templates/engage/developing-android-on-ubuntu.html]: you can add 2 background-image on the same line like this `background-image: linear-gradient(199deg, #E95420 0%, #5E2750 25%, #2C001E 50%)  url('{{ ASSET_SERVER_URL }}1487c2c6-suru-background.svg');`

**Comment by pmahnke** [comment]
@spencerbygraves and @solazio I have updated for your issues.

**Comment by spencerbygraves** [comment]
@pmahnke great, thanks.

**Pull request merged**